src/components/SharedComponents/Accordion.style.tsx

```diff
@@ -23,13 +25,21 @@ export const StyledAccordionSummary = styled(AccordionSummary)`
 
 export const MarkdownBody = styled(MarkdownContent)`
   color: ${COLOR_MAP.GRAY_BODY_COPY};
+  line-height: 1.4;
+  font-family: Roboto;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 22px;
```

Comment:
you only need to declare this block of styles once (in the `p`, as you've done below)